### DISCUSSION TITLE
Unable to delete airflow Dag history through UI

### DISCUSSION THREAD

**Discussion by RanganathLH** [apache/airflow#31051] [Q&A]
### Body

Hi Team,

We are trying to delete airflow one of the dag history through UI delete option but it give Ooops!.

We are using 2.2.2 version airflow and kindly let us know what might be causing the issue.

or is that a airflow bug ?

### Committer

- [X] I acknowledge that I am a maintainer/committer of the Apache Airflow project.

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by RanganathLH** [comment]
Here is the screenshot for the same:
![image](https://user-images.githubusercontent.com/91545788/236113729-e78f32d0-dabe-4677-b2d5-885621b50b51.png)
![image](https://user-images.githubusercontent.com/91545788/236113793-be654aee-2e03-4c21-bcc1-34d10bf82678.png)

**Comment by eladkal** [comment]
Not able to reproduce on latest Airflow.
Does this happen on specific dag or in all your dags?
It's likely that the issue will go away once you upgrade Airflow

**Comment by eladkal** [comment]
Also please do not open issues as maintainer
![Screenshot 2023-05-04 at 9 37 30](https://user-images.githubusercontent.com/45845474/236128568-6f6f14a8-04e7-4e52-ac96-ddf41f00ffd8.png)

There is a bug template to follow:
![Screenshot 2023-05-04 at 9 38 36](https://user-images.githubusercontent.com/45845474/236128675-308cca07-a2a2-46a2-9495-b275a91574b7.png)

**Reply by RanganathLH** [reply]
Hi @eladkal , Airflow installed in AWS environment and not sure whether this issue with respect to environment level or common one for 2.2.2 version. I am going to raise concern with AWS support team as well. We will give you update on this if anything comes from AWs support team. One more thing, This is happening for all the dags
